concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetFinalizedBlocks.java

```diff
@@ -0,0 +1,52 @@
+package com.concordium.sdk.examples;
+
+import com.concordium.sdk.ClientV2;
+import com.concordium.sdk.Connection;
+import com.concordium.sdk.Credentials;
+import com.concordium.sdk.exceptions.ClientInitializationException;
+import lombok.var;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetFinalizedBlocks", mixinStandardHelpOptions = true)
+public class GetFinalizedBlocks implements Callable<Integer> {
+    @Option(
+            names = {"--endpoint"},
+            description = "GRPC interface of the node.",
+            defaultValue = "http://localhost:20001")
+    private String endpoint;
+
+    @Option(
+            names = {"--timeout"},
+            description = "GRPC request timeout in milliseconds.",
+            defaultValue = "100000")
+    private int timeout;
+
+    @Override
+    public Integer call() throws MalformedURLException, ClientInitializationException {
```

Comment:
Same as above.

Reply:
Answered Above

"Call here is not being used to specify a return code. Returning an exit code is a side effect which might be useful in further development.

Currently its being used so that Checked Exceptions can be added to the Method signature. Since earlier approach to use @SneakyThrows in the Examples code was not an Elegant Solution.
The Library used to create console application automatically handles the Exceptions and prints them to the console."